docs/graphcast/design-principles.md

```diff
@@ -19,22 +19,28 @@ The SDK is the base layer which is used to abstract all the necessary components
 - Checks message validity for past message injections, nonexistent blocks and expired timestamps. It also guarantees that messages are signed by an authorised operator address of an active on-chain Indexer (this can be used as a basis for a reputation system).
 - Supports a flexible and customizable configuration of the Graphcast gossip agent, enabling specification of network settings, peer discovery mechanisms, message encoding formats, and more. For detailed instructions on configuring Graphcast to suit your needs, refer to the configuration guide.
 - Topics in Graphcast represent different categories or subjects of information. Nodes can dynamically subscribe to specific topics to receive messages related to those topics. Topics enable efficient message routing and dissemination within the network.
-- Provides comprehensive message handling structure to ensure that messages are reliably transmitted, received, and processed within the network. 
+- Provides comprehensive message handling structure to ensure that messages are reliably transmitted, received, and processed within the network.
 
 ## Radios
 
 General Radio components
 
 - Supports Radio for specific use cases.
-- Controls topic subscriptions dynamically for interested topics. 
-- Provides radio type definition used to verify the integrity and authenticity of messages exchanged within the network.
+- Controls topic subscriptions dynamically for interested topics.
+- Provides Radio type definition used to verify the integrity and authenticity of messages exchanged within the network.
 - Collects Radio-specific information and incorporates it into Graphcast messages along with other relevant metadata.
 - Observes and handles relevant messages received from peers.
 - Provides performance metrics, logs, and API services.
 
-Our first example Radio is built for real-time cross-checking of Indexer Proof of Indexing attestations (POIs). Indexers must generate valid POIs to earn indexing rewards. Indexers find it beneficial to alert each other on the health status of subgraphs in community discussions. To alleviate the manual workload, the POI Radio:
+The first Radio built on top of Graphcast is the Subgraph Radio. It's designed to facilitate real-time information exchange among participants in The Graph network and serves as a tool for Indexers and other network participants to share valuable Subgraph data.
+
+With Subgraph Radio, Indexers can run a single Radio instance and track a wide variety of message types and data related to Subgraphs. Different use cases and message types form the different _functionalities_ of the Radio.
+
+For now the Radio supports only one functionality - the Proof of Indexing (POI) functionality.
```

Comment:
Maybe update "the Proof of Indexing functionality" to "the Proof of Indexing crosschecking". Also, the radio does technically support version upgrade messages. It would be good to include it to accurately reflect the functionalities even if it simply sends off a notification; it makes the Subgraph radio scope more solid, but I'm fine if you would like to include that later on.